dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -100,36 +99,26 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
 
     winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationBuilder AppNotificationBuilder::SetAttributionText(hstring const& text)
     {
-        THROW_HR_IF(E_INVALIDARG, !m_attributionText.empty());
-
         m_attributionText = wil::str_printf<std::wstring>(L"<text placement='attribution'>%ws</text>", text.c_str());
-
         return *this;
     }
 
     winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationBuilder AppNotificationBuilder::SetAttributionText(hstring const& text, hstring const& language)
     {
-        THROW_HR_IF(E_INVALIDARG, !m_attributionText.empty());
-        THROW_HR_IF(E_INVALIDARG, language.empty());
+        THROW_HR_IF_MSG(E_INVALIDARG, language.empty(), "You must provide a language calling SetAttributionText.");
```

Comment:
NIT: Remove the period. Don't need pretty full english sentence for an error message. If it were a paragraph of sentences then yes, but for a single sentence/phase, nah

same comment applies throughout

Reply:
I wasn’t sure to add a period or not. Will fix.

Reply:
got a PR to address this last round of feedback, here: https://github.com/microsoft/WindowsAppSDK/pull/2801

Reply:
Addressed in https://github.com/microsoft/WindowsAppSDK/pull/2801